vars/visualisingTest.groovy

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env groovy
```

Comment:
Still not clear why this wrapper is needed ATM

Reply:
Removed it as it is not used